packages/core/src/components/tooltip/tooltip.md

```diff
@@ -1,7 +1,45 @@
 @# Tooltip
 
-`Tooltip` simply passes its props to [`Popover`](#labs/popover) with some modified defaults. The public API is the same as [`Tooltip`](#core/components/tooltip), except for the changes noted in the `Popover` documentation.
+`Tooltip` simply passes its props to [`Popover`](#labs/popover) with some exceptions.
+Notably, it only supports `HOVER` interactions and the `target` prop is not supported.
 
 @reactExample TooltipExample
 
+@## JavaScript API
+
+The `Tooltip` component is available in the __@blueprintjs/core__ package.
+Make sure to review the [general usage docs for JS components](#blueprint.usage).
+
+When creating a tooltip, you must specify both:
+- its _content_, by setting the `content` prop, and
+- its _target_, as a single child element or string.
+
+The content will appear in a contrasting popover when the target is hovered over.
+
+<div class="pt-callout pt-intent-warning pt-icon-warning-sign">
+    <h4 class="pt-callout-title">Button targets</h4>
+    Buttons make great tooltip targets, but the `disabled` attribute will prevent all
+    events so the enclosing `Tooltip` will not know when to respond.
+    Use [`AnchorButton`](#core/components/button.anchor-button) instead;
+    see the [callout here](#core/components/button.javascript-api) for more details.
+</div>
+
 @interface ITooltipProps
+
+@## Combining with popover
```

Comment:
nit: pick one of these
- Combining with `Popover`
- Combining with a popover
- Combining with popovers